addons/l10n_it_edi_withholding/__init__.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import models
+from odoo import api, SUPERUSER_ID
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def _l10n_it_edi_add_accounts(env, company):
+    account_codes = ['1611', '2603']
+    for account_code in account_codes:
+        template_vals = []
+        if not env['account.account'].search([('code', '=', account_code)]):
+            account_template = env.ref(f'l10n_it_edi_withholding.{account_code}')
+            vals = company.chart_template_id._get_account_vals(company, account_template, account_code, {})
+            template_vals.append((account_template, vals))
+        company.chart_template_id._create_records_with_xmlid('account.account', template_vals, company)
+    _logger.info("Created withholding accounts")
+
+
+def _l10n_it_edi_withholding_add_taxes(env, company):
+    # Create the new taxes on existing company
+    existing_taxes = no_tax = env['account.tax']
+    templates = env['account.tax.template']
+    for xml_id in (
+        '20awi', '20vwi',
+        '20awc', '20vwc',
+        '23awo', '23vwo',
+        '4vcp', '4acp',
+        '4vinps', '4ainps'
+    ):
+        templates |= env.ref(f"l10n_it_edi_withholding.{xml_id}")
+        tax = env.ref(f"l10n_it_edi_withholding.{company.id}_{xml_id}", raise_if_not_found=False) or no_tax
+        existing_taxes |= tax
+    if not existing_taxes:
+        templates._generate_tax(company)
+        _logger.info("Created withholding taxes")
+
+    # Increase the sequence number of the old taxes multiplying it by 10 and adding 21
+    # so that the withholding can have sequence=10 and the pension fund sequence=20
+    offset = 21
+    all_taxes = env['account.tax'].search([('company_id', '=', company.id)])
+    for tax in all_taxes.filtered(lambda x: (x.sequence <= 20 and not x.l10n_it_pension_fund_type and not x.l10n_it_withholding_type)):
+        if not tax.l10n_it_withholding_type and not tax.l10n_it_pension_fund_type:
+            tax.sequence += offset
```

Comment:
@lordkrandel Something has been forgotten here.  The Cash Transition Account is not set.  
![Screenshot from 2023-01-05 16-31-41](https://user-images.githubusercontent.com/3832699/210818334-e73169a0-a43e-4b4e-8b50-61c8ea6f940b.png)

https://github.com/odoo/odoo/blob/c44a33d67065e20c4ded2a229d0663191c91a0a0/addons/account/models/chart_template.py#L748

Reply:
I wonder what's happening here... the cash_basis_transition_account is on the tax (the new companies have it).
I'll start debugging.

https://github.com/odoo/odoo/blob/c44a33d67065e20c4ded2a229d0663191c91a0a0/addons/account/models/chart_template.py#L1328